WORKSPACE

```diff
@@ -35,16 +34,10 @@ git_repository(
 # CUDA should be installed on the system locally
 new_local_repository(
     name = "cuda",
-    path = "/usr/local/cuda-11.1/",
+    path = "/usr/local/cuda",
     build_file = "@//third_party/cuda:BUILD",
 )
 
-new_local_repository(
```

Comment:
I though cuBLAS was a dependency for TensorRT, do we not need this?

Reply:
Nope - yes cuBLAS is a dependency, but we have cuda:cublas target for it. As you can see, this one was not referred to from any BUILD file.